src/praisonai/praisonai/framework_adapters/praisonai_adapter.py

```
@@ -56,18 +53,73 @@ def run(
         agents = {}
         tasks = []
+        tasks_dict = {}
+        
+        # Use tool resolver instance for local tools.py integration if available
+        tools_list = []
+        try:
+            # Access the tool_resolver if it exists (from the dead code pattern)
+            if hasattr(cli_config, 'tool_resolver') and cli_config.tool_resolver:
+                tools_list = cli_config.tool_resolver.get_local_callables()
+                logger.debug(f"Loaded tools from tools.py: {tools_list}")
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)